sechub-webui/src/test/java/com/mercedesbenz/sechub/webui/page/LoginOAuth2ControllerTest.java

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.webui.page;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.context.annotation.Import;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.web.servlet.MockMvc;
+
+import com.mercedesbenz.sechub.webui.YamlPropertyLoaderFactory;
+import com.mercedesbenz.sechub.webui.security.SecurityTestConfiguration;
+
+@WebMvcTest(LoginOAuth2Controller.class)
+@Import(SecurityTestConfiguration.class)
+@ActiveProfiles("oauth2-enabled")
+@TestPropertySource(locations = "classpath:application-test.yml", factory = YamlPropertyLoaderFactory.class)
+class LoginOAuth2ControllerTest {
+
+    private final MockMvc mockMvc;
+
+    @Autowired
+    LoginOAuth2ControllerTest(MockMvc mockMvc) {
+        this.mockMvc = mockMvc;
+    }
+
+    @Test
+    void login_o_auth_2_page_is_accessible_anonymously() throws Exception {
```

Comment:
Should we test also (in another test method) that login in classic mode is not supported in this case?